tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/PushDataTimeoutTest.scala

```diff
@@ -66,27 +100,23 @@ class PushDataTimeoutTest extends AnyFunSuite
       val rssSparkSession = SparkSession.builder()
         .config(updateSparkConf(sparkConf, ShuffleMode.HASH))
         .getOrCreate()
-      val rssCombineResult = combine(rssSparkSession)
-      val rssGroupbyResult = groupBy(rssSparkSession)
-      val rssRepartitionResult = repartition(rssSparkSession)
       val rssSqlResult = runsql(rssSparkSession)
 
-      assert(combineResult.equals(rssCombineResult))
-      assert(groupbyResult.equals(rssGroupbyResult))
-      assert(repartitionResult.equals(rssRepartitionResult))
-      assert(combineResult.equals(rssCombineResult))
       assert(sqlResult.equals(rssSqlResult))
 
       rssSparkSession.stop()
       ShuffleClient.reset()
+      assert(PushDataHandler.pushMasterMergeDataTimeoutTested.get())
+      if (enabled) {
+        assert(PushDataHandler.pushSlaveMergeDataTimeoutTested.get())
+      }
     }
   }
 
   test("celeborn spark integration test - pushdata timeout will add to balcklist") {
     val sparkConf = new SparkConf().setAppName("rss-demo").setMaster("local[2]")
       .set(s"spark.${CelebornConf.CLIENT_PUSH_DATA_TIMEOUT.key}", "5s")
-      .set(s"spark.celeborn.data.push.timeoutCheck.interval", "2s")
-      .set(s"spark.${CelebornConf.CLIENT_BLACKLIST_SLAVE_ENABLED.key}", "false")
+      .set(s"spark.${CelebornConf.CLIENT_BLACKLIST_SLAVE_ENABLED.key}", "true")
```

Comment:
this should be accidentally changed by https://github.com/apache/incubator-celeborn/pull/1501